### PR TITLE
Add automated Android installation scripts on Windows

### PR DESCRIPTION
**Pull request by patelneel55**

### CODE REVIEW

File: install-sdk-packages.ps1 [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+""
+
+# Install all packages
+foreach($package in $PACKAGES)
+{
+	Write-Host "Installing $package..."
+
+	sdkmanager $package
+
+	if($LastExitCode -ne 0)
```
Comment (jmcker) [inline install-sdk-packages.ps1:50]: Should the other packages be able to finish if one fails?

Comment (jmcker) [reply install-sdk-packages.ps1:50]: I see the Linux one exits the same way. Maybe we should change both

File: install-sdk-packages.ps1 [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+# Check for SDK Home
+if(-NOT (Test-Path -path $env:ANDROID_HOME))
+{
+	Write-Host "ANDROID_HOME not defined" -Foreground Red
+	""
+	Write-Host "Assuming SDK is at $env:LOCALAPPDATA\Android\sdk" -Foreground Yellow
+	""
+	[Environment]::SetEnvironmentVariable("ANDROID_HOME", "$env:LOCALAPPDATA\Android\sdk", "Machine")
```
Comment (jmcker) [inline install-sdk-packages.ps1:28]: Is this ok at just `"Machine"`? I see `[EnvironmentVariableTarget]::Machine` other places

**Review by jmcker — APPROVED** [review]

File: install-sdk-packages.ps1 (hunk 2 above)
Comment (patelneel55) [reply install-sdk-packages.ps1:28]: Yea it's one or the other...I kinda did both didn't I😂😂

File: install-sdk-packages.ps1 (hunk 1 above)
Comment (patelneel55) [reply install-sdk-packages.ps1:50]: I see what you mean....but how would you know what package failed if you keep going? I guess we could keep track of the package failed when code is not 0

**Pull request closed without merging**